editor/editor_node.cpp

```diff
@@ -197,6 +197,9 @@ void EditorNode::_version_control_menu_option(int p_idx) {
 
 			VersionControlEditorPlugin::get_singleton()->shut_down();
 		} break;
+		default: {
+			ERR_PRINTS("No such VCS menu handler!");
```

Comment:
I was thinking about displaying a graphical error dialog:

```cpp
EditorNode::get_singleton()->show_warning(TTR("No such VCS menu handler. This is a bug, please report."));
```

This way, users who didn't start Godot in a terminal will be able to know what's going on :slightly_smiling_face:

Reply:
I was thinking about too. Need users know about such errors (regressions)? But as a way... Although, why not.